publisher-sdk/src/androidTest/java/com/criteo/publisher/integration/DfpHeaderBiddingFunctionalTest.java

```diff
@@ -672,11 +691,34 @@ public void onAdLoaded() {
       }
 
       @Override
-      public void onAdFailedToLoad(int i) {
+      public void onAdFailedToLoad(LoadAdError loadAdError) {
+        LoggerFactory.getLogger(DfpHeaderBiddingFunctionalTest.class)
+            .debug("onAdFailedToLoad for reason %d. "
+                    + "See: https://developers.google.com/android/reference/com/google/android/gms/ads/doubleclick/PublisherAdRequest",
```

Comment:
The URL can be removed. It was added as a pointer to the doc to explain the meaning of the `int` from old listener. Now that the listener use a proper `LoadAdError` enum, I guess it is no longer needed.

Reply:
done